src/libraries/System.IO.FileSystem/tests/File/GetSetTimes.cs

```diff
@@ -121,14 +286,88 @@ public void PageFileHasTimes()
             string pageFilePath = Directory.EnumerateFiles(@"C:\", "pagefile.sys").FirstOrDefault();
             if (pageFilePath != null)
             {
-                Assert.All(TimeFunctions(), (item) =>
+                // Due to the fact that the pagefile.sys is already opened by the operating system (ntoskrnl.exe - PID: 4),
+                // we exclude the `SafeFileHandle`-`TimeFunctions`, because the handle cannot be opened twice.
+                Assert.All(TimeFunctions(false, false), item =>
                 {
                     var time = item.Getter(pageFilePath);
                     Assert.NotEqual(DateTime.FromFileTime(0), time);
                 });
             }
         }
 
+        [Fact]
+        public void GetCreationTime_SafeFileHandle_Conformance()
```

Comment:
Do these `*_Conformance` tests increasing coverage compared to what is already performed by `TimeFunctions`?

Reply:
No they do not. I added them to verify that GetCreationTime (with path) returns the same result as the SafeFileHandle overload.
But after thinking about it now, I realized that this is not useful. I'll remove that in a moment.